plugin/webhook/webhook.go

```diff
@@ -86,6 +86,14 @@ func (c *Context) getMetaList() []meta {
 			Name:  "Issue",
 			Value: c.Issue.Name,
 		})
+		description := c.Issue.Description
```

Comment:
Any reason that we append a new meta instead of updating the value above? Or add a comment to explain it.

Reply:
Added comments.